packages/synapse-react-client/src/components/MissingQueryResultsWarning/MissingQueryResultsWarning.tsx

```diff
@@ -26,9 +26,9 @@ export type MissingQueryResultsWarningProps = {
 const MissingQueryResultsWarning: React.FunctionComponent<
   MissingQueryResultsWarningProps
 > = ({ entity }) => {
-  // Currently, Datasets are the only table type for which we can reliably get this info.
+  // Currently, Datasets and Dataset collections are the only table type for which we can reliably get this info.
   // Other cases will need a new service, tracked by PLFM-7046
-  const isMissingResultsCalculable = entity && isDataset(entity)
+  const isMissingResultsCalculable = entity && isEntityRefCollectionView(entity)
```

Comment:
Made a few slight improvements while investigating PLFM-8326 (we initially thought it was a frontend issue)